apps/asap-cli/src/import/protocols/parse.ts

```diff
@@ -16,13 +16,31 @@ export default (data: string[]): Protocol => {
     link,
     name,
     version,
-    authors,
     created,
-    contact,
+    authors,
     keywords,
+    owner,
     abstract,
   ] = data.map((s) => s.trim());
 
+  const [day, month, yearAndTime] = created.split('.');
+
+  let abstractText;
+
+  // try to parse string as JSON or fail gracefully
+  try {
+    abstractText = ['(private protocol)', 'Abstract crypted'].includes(abstract)
```

Comment:
i think you're try/catching a little bit more than just JSON.parse here. Are you sure you want to do that?

Reply:
What else would make it throw here?

Reply:
reduce, concat, includes

Reply:
Yeah, no worries, I'm fine with this failing silently.